pkgs/os-specific/linux/rasdaemon/default.nix

```
@@ -0,0 +1,102 @@
+{ stdenv, lib, fetchurl, automake, autoreconfHook, coreutils, dmidecode
+, glibcLocales, hwdata, kmod, libtool, makeWrapper, sqlite, perl, doCheck ? true
```

```suggestion
, glibcLocales, hwdata, kmod, libtool, makeWrapper, sqlite, perl
```

It's not necessary, can be overridden with `.overrideAttr`.